portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +34,35 @@ function createCommentElement(userName, comment) {
   divElement.innerText = userName + ': ';
   divElement.appendChild(pElement);
   return divElement;
-}
\ No newline at end of file
+}
+
+function getLoggedInStatus() {
+  fetch('/data').then(response => response.json()).then((json) => {
+    const loggedIn = json.loggedInStatus;
+
+    if (loggedIn){
+      const pElement = document.getElementById('display-user');
+      pElement.style.display = "block";
+      pElement.textContent = "Logged in as: " + json.userEmail;
+      document.getElementById('logout-link').href = json.logoutUrl;
+      document.getElementById('comment-form').style.display = "block";
+    }else
+      createLoginButton(json.loginUrl);
+  });
+}
+
+function createLoginButton(loginUrl) {
```

Comment:
Can you do the button creation etc in html and only have this method set the href?

Reply:
Another option is to redirect them to a login page and not have a button in the main page.

Reply:
Done: 60eacb3 
- add button in html